bedrock/settings/static_media.py

```diff
@@ -1211,18 +1176,11 @@
     'firefox_firstrun': {
         'source_filenames': (
             'js/base/mozilla-fxa-iframe.js',
+            'js/base/uitour-lib.js',
```

Comment:
I assume this is necessary, but I'm missing what for.

Reply:
Hmm, at one time it was needed for the end page transition, but now it goes to `about:home` it may not be needed. I'll check 👍